src/components/FindMovie/FindMovie.tsx

```diff
@@ -20,38 +73,84 @@ export const FindMovie: React.FC = () => {
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {error && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
```

Comment:
use interpolation here
```suggestion
              {`Can't find a movie with such a title`}
```

Reply:
сразу же ругается линтер на кавычки. Пришлось отключить правило для одиночных кавычек